### PR TITLE
3436 clean-up PM launcher editable_select interface

### PR DESCRIPTION
**Pull request by euler-room**
Moved `add` & `remove` buttons below the `choice` label in launcher's `_editable_select` partial.

### CODE REVIEW

**Comment by euler-room** [comment]
Image 10-21-24 at 8 04 AM

**Comment by euler-room** [comment]
This was my reasoning behind the change:

Screenshot 2024-10-21 at 9 11 09 AM

The line break is very odd looking.
Even if I forced the `<button><choice><button>` to remain on the same line, it would look strange with the buttons so far apart.

**Comment by euler-room** [comment]
Screenshot 2024-10-21 at 9 32 22 AM
Screenshot 2024-10-21 at 9 33 04 AM

I reworked the code and have it operational.

File: apps/dashboard/app/views/launchers/editable_form_fields/_editable_select.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <%= render(partial: 'launchers/editable_form_fields/edit_fixed_field',  locals: { form: form, attrib: attrib, fixed: fixed }) %>
 
-    <ol class="list-group text-center col-md-4 mb-3">
+    <ol class="list-group list-group-flush text-center col-md-4 mb-3">
```
Comment (johrstrom) [inline apps/dashboard/app/views/launchers/editable_form_fields/_editable_select.html.erb:15]: I don't know if I like the `list-group-flush` here - it seems off to me to only have borders on the bottom/top. 

It seems like we should actually just deal with the overflow in the span right?

Comment (johrstrom) [reply apps/dashboard/app/views/launchers/editable_form_fields/_editable_select.html.erb:15]: Also these can `flex` to grow (to a certain size, maybe ~33%?) but it does seem like we should actually just deal with `text-overflow` correctly with potential resizing.

**Comment by euler-room** [comment]
https://github.com/user-attachments/assets/e239257f-6219-4a98-b140-d519dc141652

**Pull request merged**